commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliAzureKeyVaultParameters.java

```diff
@@ -61,6 +64,14 @@ public class PicoCliAzureKeyVaultParameters implements AzureKeyVaultParameters {
       paramLabel = "<CLIENT_SECRET>")
   private String clientSecret;
 
+  @Option(
+      names = {"--azure-secrets-tags"},
+      description =
+          "Specify optional tags to filter the secrets. "
+              + "For example --azure-secrets-tags ENV=prod --azure-secrets-tags=CLIENT=A.",
+      paramLabel = "<TAG_NAME=TAG_VALUE>")
```

Comment:
AWS implements --aws-secrets-tag-names-filter and --aws-secrets-tag-values-filter
so this is inconsistent from a UX perspective, though not sure how much that matters in this instance.

Is there a reason this was implemented differently?